src/actions/actionProperties.tsx

```diff
@@ -674,24 +677,35 @@ export const actionChangeArrowhead = register({
   PanelComponent: ({ elements, appState, updateData }) => (
     <fieldset>
       <legend>{t("labels.arrowheads")}</legend>
-      <div className="buttonList buttonListIcon">
-        <ButtonIconCycle
-          group="arrowhead_start"
+      <div className="iconSelectList">
```

Comment:
I think it might make sense to reverse the two arrowhead pickers in RTL mode so they still “connect:”
<img width="107" alt="Screenshot_2020-12-11 10 27 49" src="https://user-images.githubusercontent.com/25517624/101921963-8f97ec00-3b9b-11eb-99ed-33c3ecb157f1.png"><img width="115" alt="Screenshot_2020-12-11 10 27 57" src="https://user-images.githubusercontent.com/25517624/101921982-93c40980-3b9b-11eb-9768-c7798fd4d9d2.png">

Reply:
Great catch. The shapes should now look correct in RTL languages.